components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1527,9 +1530,17 @@ void RewardsServiceImpl::GetAddresses(const GetAddressesCallback& callback) {
   if (!Connected()) {
     return;
   }
+  int32_t current_country(
```

Comment:
nit: could you please do `int32_t current_country = country_codes::GetCountryIDFromPrefs(profile_->GetPrefs())` as it's a lot easier to read

Reply:
Done.